cnames_active.js

```diff
@@ -1831,6 +1831,7 @@ var cnames_active = {
   "mc": "magicarbon.github.io/mc", // noCF? (don´t add this in a new PR)
   "mca": "lem0nb.github.io/Minecraft_Auto_Builder",
   "mcd": "n9gc.github.io/mcdjs",
+  "mcfly": "mc-fly.vercel.app",
```

Comment:
```suggestion
  "mcfly": "cname.vercel-dns.com", // noCF
```
As the CI notes (as does https://vercel.com/docs/projects/domains/add-a-domain#subdomains), you'll want to point at `cname.vercel-dns.com`. You'll also want to disable the Cloudflare proxy with `// noCF` as Vercel and Cloudflare do not mix well.

Reply:
Thanks!!